pkg/base1/cockpit.js

```diff
@@ -1632,7 +1632,6 @@ function full_scope(cockpit, $, po) {
                 owner = msg.owner;
             } else {
                 console.warn("received unexpected dbus json message:", payload);
```

Comment:
This isn't an error. Perhaps worth a debug line, but not a ```console.warn```. Isn't it expected that new messages will be added to ```dbus-json3```? That's the intent of this change, right?